src/sentry/migrations/0185_rm_copied_useroptions.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 1.11.29 on 2021-04-07 21:30
+
+from django.db import migrations
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def delete_duplicate_useroption_rows(apps, schema_editor):
+    """
+    Delete the rows in UserOption that have already been copied over
+    to the NotificationSetting table, as well as a few unused ones while we're at it
+    """
+    UserOption = apps.get_model("sentry", "UserOption")
+    for user_option in RangeQuerySetWrapperWithProgressBar(UserOption.objects.all()):
+        if user_option.key in (
+            "workflow:notifications",
+            "mail:alert",
+            "deploy-emails",
+            "subscribe_by_default",
+            "seen_release_broadcast",
+            "twilio:alert",
+            "workflow_notification",
```

Comment:
Should this list be conditions to the query?

Reply:
We usually avoid this when using the ranged wrapper. Since we're already sorting by id, no indexes can be used anyway.